packages/@aws-cdk/aws-servicecatalog/lib/constraints.ts

```diff
@@ -62,4 +63,41 @@ export interface TagUpdateConstraintOptions extends CommonConstraintOptions {
    * @default true
    */
   readonly allow?: boolean;
+}
+
+/**
+ * An assertion within a template rule, defined by intrinsic functions.
+ */
+export interface Assertion {
```

Comment:
This interface's name is way too generic.

It should be called something like `TemplateRuleAssertion`.